scad/parameters.scad

```diff
@@ -144,7 +147,12 @@ h_unit = unit;
 // Vertical unit size (17mm for choc keycaps)
 v_unit = unit;
 // Spacing of grid for MX pins
-grid = 1.27;
+grid =
+    switch_type == "mx"
+    ? 1.27
+    : switch_type == "choc"
+        ? 0
+        : assert(false, "switch_type is invalid");
```

Comment:
Good point. Pretty sure I didn't make this variable(I think?), but happy to refactor while I'm there. Better double-check it isn't being added to anything without a mx check

Reply:
yeah I made this variable and was planning to rename it since `grid` is quite ambiguous. Then I saw your PR and wanted to make sure you weren't relying on it for anything.

Reply:
I'll rename it when i adress these other things